docs/source/integrations/databricks_deployment_workflow.md

```diff
@@ -0,0 +1,316 @@
+    with KedroSession.create(env=env, conf_source=conf_source) as session:
+        session.run()
+
+
+if __name__ == "__main__":
+    main()
+```
+
+2. **Define a new entry point**: Open `<project_root>/src/setup.py` in a text editor or IDE and add a new line in the definition of the `entry_point` tuple, so that it becomes:
+
+```python
+entry_point = (..., "databricks_run = <package_name>.databricks_run:main")
+```
+
+Remember to replace <package_name> with the correct package name for your project.
+
+This process adds an entry point to your project which can be used to run it on Databricks.
+
+```{note}
+Because you are no longer using the default entry-point for Kedro, you will not be able to run your project with the options it usually provides. Instead, the `databricks_run` entry point in the above code and in the PySpark Iris starter contains a simple implementation of two options:
+- `--env`: specifies a configuration environment to load for your run.
+- `--conf-source`: specifies the location of the `conf/` directory to use with your Kedro project.
+```
+
+### Plan your project's output
+
+You will not be able to directly see your project's log output when it is run as a Databricks job. You will need to decide on and implement a strategy for storing your project's results. Some examples are:
+
+- Write the result of a data engineering pipeline to a managed delta table.
+- Upload metrics from a machine learning model to MLflow.
+- Save a dataset as a CSV file in DBFS.
+
+In this guide, your project will save its logs to a location in DBFS.
+
+- **Create a directory to save your logs into on DBFS**: at the command line in your local environment, run the following Databricks CLI command:
+
+```bash
+databricks fs mkdirs dbfs:/FileStore/iris-databricks/logs
+```
+
+This command creates a directory on DBFS (`/dbfs/FileStore/iris-databricks/logs`) where your project can save its logs.
+
+**The `databricks` environment of the PySpark Iris starter is already configured to save logs in this location.** The configuration used to achieve this is part of the [`databricks` environment](../configuration/configuration_basics.md#configuration-environments) defined for PySpark Iris and can be found in the `conf/databricks/logging.yml` file.
+
+### Package your project
+
+To package your Kedro project for deployment on Databricks, you must create a Wheel (`.whl`) file, which is a binary distribution of your project. In the root directory of your Kedro projec   t, run the following command:
+
+```bash
+kedro package
+```
+
+This command generates a `.whl` file in the `dist` directory within your project's root directory.
+
+### Upload project data and configuration to DBFS
+
+Your packaged Kedro project needs access to data and configuration in order to run. Therefore, you will need to upload your project's data and configuration to an accessible location. In this guide, we will store the data on the Databricks File System (DBFS).
```

Comment:
```suggestion
Your packaged Kedro project needs access to data and configuration in order to run. Therefore, you will need to upload your project's data and configuration to a location accessible to Databricks. In this guide, we will store the data on the Databricks File System (DBFS).
```

Reply:
Are there other options besides DBFS? Might be worth listing them (and pointing to links to find out more?) if so.

Reply:
Hmm, I like this idea, but I feel a problem in this case is that there are so many possible ways to store config and data in an internet-accessible location and these have so many caveats. If I start an explanation here there's no way to quickly finish it without leaving the user with more questions and showing them a red herring. I would prefer to focus the user's attention on Databricks in this guide, as this is already a lot to take in at once.

If you'd like I can change the wording here to make it less suggestive that there are other ways of achieving this.